Dockerfile.redhat

```diff
@@ -127,19 +130,74 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
             unzip \
             vim \
             xz \
-            https://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/tbb-2018.2-9.el8.x86_64.rpm && \
+            https://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/tbb-2018.2-9.el8.x86_64.rpm \
+            http://mirror.centos.org/centos/8-stream/PowerTools/x86_64/os/Packages/libusb-devel-0.1.5-12.el8.x86_64.rpm \
+            http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/libusb-0.1.5-12.el8.x86_64.rpm \
+            http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/libusbx-devel-1.0.23-4.el8.x86_64.rpm && \
             yum clean all
 
+ARG NVIDIA=0
+# Add Nvidia dev tool if needed
+# hadolint ignore=DL3003
+RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
+    yum config-manager --save --set-enabled codeready-builder-for-rhel-8-x86_64-rpms && \
+    yum -y module disable python36 && \
+    yum install -y  \
+        libzstd-devel \
+        libcudnn8-8.6.0.163-1.cuda11.8 \
+        libcudnn8-devel-8.6.0.163-1.cuda11.8 \
+        libcutensor1-1.6.1.5-1 \
+        libcutensor-devel-1.6.1.5-1 \
+        cuda-cudart-devel-11-8 && \
+    # ignore errors on hosts with older nvidia drivers
+    yum install -y cuda-11-8 || true && \
```

Comment:
Is it intentional? 
`|| true`

Reply:
@dtrawins

Reply:
Yes, that is intentional for the case with older driver installed on the host causing conflicts on some on the nested dependencies